components/pedigree/resources/src/main/resources/pedigree/view/printDialog.js

```diff
@@ -236,7 +217,7 @@ define([
                                                                     this._moveHorizontally,
                                                                     options);
             this.previewContainer.update(previewHTML);
-            this._adjustPreviewWindowHeight();
+            GraphicHelpers.adjustPreviewWindowHeight("pedigree-print-dialog", 'printPreview', this._minPreviewHeight, this._maxPreviewHeight);
```

Comment:
to avoid repetition of arguments `"pedigree-print-dialog", 'printPreview', this._minPreviewHeight, this._maxPreviewHeight"` I would keep `this._adjustPreviewWindowHeight()` method, and internally just called `GraphicHelpers.adjustPreviewWindowHeight`

Same with exportSelector.js (where i would add a new `_adjustExportWindowHeight()`)